### ISSUE TITLE
Sealed values and java conversion produce code that doesn't compile

### ISSUE THREAD

**Issue by thesamet** [scalapb/ScalaPB#1158]
When java conversions are enabled, the following protos produce a code that doesn't compile:

```
syntax = "proto3";

package app.proto;

message Event {
  oneof sealed_value {
    MessageX msgX = 1;
    MessageY msgY = 2;
  }
}

message MessageX {
  repeated string ids = 1;
}

message MessageY {
  string id = 1;
}
```

Tested with ScaslaPB 0.10.11 and 0.11.3. The compiler error:

```
sbt:scalapb-example> compile
[info] Compiling 1 protobuf files to /tmp/scalapb-example/target/scala-2.13/src_managed/main
[info] compiling 2 Scala sources and 1 Java source to /tmp/scalapb-example/target/scala-2.13/classes ...
[error] /tmp/scalapb-example/target/scala-2.13/src_managed/main/app/proto/events/Event.scala:274:43: value asJava is not a member of Seq[String]
[error]     javaPbOut.addAllIds(scalaPbSource.ids.asJava)
[error]                                           ^
[error] /tmp/scalapb-example/target/scala-2.13/src_managed/main/app/proto/events/Event.scala:278:35: value asScala is not a member of com.google.protobuf.ProtocolStringList
[error]     ids = javaPbSource.getIdsList.asScala.iterator.map(_root_.scala.Predef.identity).toSeq
[error]                                   ^
[error] /tmp/scalapb-example/target/scala-2.13/src_managed/main/app/proto/events/Event.scala:278:76: missing argument list for method identity in object Predef
[error] Unapplied methods are only converted to functions when a function type is expected.
[error] You can make this conversion explicit by writing `identity _` or `identity(_)` instead of `identity`.
[error]     ids = javaPbSource.getIdsList.asScala.iterator.map(_root_.scala.Predef.identity).toSeq
[error]                                                                            ^
[error] three errors found
[error] (Compile / compileIncremental) Compilation failed
[error] Total time: 0 s, completed May 25, 2021, 8:44:33 AM
```

Apparently, the import for Java conversions is not emitted in this case.

**Comment by juliano** [comment]
[Minimal reproducible example here.](https://github.com/juliano/scalapb-example)

**Comment by juliano** [comment]
Thank you!

**Comment by juliano** [comment]
Hi @thesamet, thanks to put the time to look into the issue. I gave the new version (0.11.4) a try and unfortunately the issue is still there. I updated my example to reproduce the error.

**Comment by thesamet** [comment]
Hi Juliano, your proejct still uses an old (0.10.x) code generator in [here](https://github.com/juliano/scalapb-example/blob/a093784410ef269bf753a0ba58bc705662bba438/project/plugins.sbt) - this needs to be updated to 0.11.4. Note that google-common-protos is not available yet for ScalaPB 0.11.x, the issue is [tracked here](https://github.com/scalapb/ScalaPB/issues/1122).

**Comment by juliano** [comment]
That makes a lot of sense 😅 

Thanks again @thesamet !
